ivadomed/transforms.py

```diff
@@ -423,6 +433,19 @@ def undo_transform(sample):
         return sample
 
 
+class BackgroundClass(mt_transforms.MTTransform):
```

Comment:
Great!
Question --> do you exclude this class when calling `CombinedDiceLoss` or rather just after the softmax?

Reply:
I don't use it anymore since I exclude the background class in the models. I let it in this PR, because I thought it might be useful in the future if we want to include the background as a class and measure different metrics on it. But since I don't use it currently, it might pollute the transform files, opinion?

Reply:
Yes, I would agree! There is no planet B.